scheduler/scheduler-api/src/main/java/org/ow2/proactive/scheduler/common/Page.java

```diff
@@ -0,0 +1,45 @@
+package org.ow2.proactive.scheduler.common;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Page<T extends Serializable> implements Serializable {
+    
+    private int size;
+
+    private ArrayList<T> list;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) The type of the "list" object should be an interface such as "List" rather than the implementation "ArrayList". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1319)